README.md

```diff
@@ -1,2 +1,31 @@
-# advisory-group
-Enterprise Advisory Group
+# Enterprise Advisory Group
+
+The Enterprise Advisory Group is a group of corporate representatives in roles of leadership responsible managing for large-scale deployments of Node.js in an Enterprise environment. As a group, we strive to provide timely feedback to the [TSC](https://github.com/nodejs/TSC), the [CommComm](https://github.com/nodejs/community-committee), the [Node.js project](https://github.com/nodejs/node) and the Node.js Foundation about the unique needs surrounding running Node.js in an Enterprise environment.
+
+This group is a part of the [Node.js End User Feedback Inititative](https://github.com/nodejs/user-feedback) which is under the charter of the [Node.js Community Committee (CommComm)](https://github.com/nodejs/community-committee). Thank you for your interest and support. Please [create an issue](https://github.com/nodejs/user-feedback/issues) in this repo if you are interested in dedicating 5+ hours a month to during the formation process.
+
+### Governance and Contributing
+The Enterprise Advisory Group has adopted the core governance and contributing policies of Node.js. This group is a part of the [Node.js End User Feedback Inititative](https://github.com/nodejs/user-feedback) which is under the charter of the [Node.js Community Committee (CommComm)](https://github.com/nodejs/community-committee).
```

Comment:
Might be good to link to the "core governance and contributing policies of Node.js". TBH, I don't think these things actually exist in any way form that would be reasonably applicable to this group. This seems a bit hand-wavy but governance and policies are things you really don't want to be hand-wavy about

Reply:
@Trott This language comes from the Evangelism WG charter, which at present is the only fully chartered Working Group under the CommComm: https://github.com/nodejs/evangelism#governance-and-contributing